schema.graphql

```diff
@@ -96,6 +96,21 @@ type ConnectionPageInfo {
   startCursor: String
 }
 
+
+input CreateAdvertisementInput {
+  endDate: Date!
+  file: String!
+  name: String!
+  organizationId: ID!
+  startDate: Date!
+  type: String!
+}
+
+type CreateAdvertisementPayload {
+  _id: ID
+  advertisement: Advertisement!
+}
+
```

Comment:
There is no need to create a separate type for it as `Advertisement` with an `_id` field as Mongoose supplies it.